test/experimental/features/eventtype_autocreation/feature.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventtype_autocreation
+
+import (
+	"context"
+	"embed"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+	eventingv1beta2 "knative.dev/eventing/pkg/apis/eventing/v1beta2"
+	eventingclient "knative.dev/eventing/pkg/client/injection/client"
+	"knative.dev/reconciler-test/pkg/environment"
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/k8s"
+	"knative.dev/reconciler-test/pkg/manifest"
+)
+
+//go:embed eventtype.yaml
+var yaml embed.FS
+
+type EventType struct {
+	Name       string
+	EventTypes func(etl eventingv1beta2.EventTypeList) (bool, error)
+}
+
+func WaitForEventType(eventtype EventType, timing ...time.Duration) feature.StepFn {
```

Comment:
Instead of reimplementing this, could you reference the one from `knative.dev/eventing/test/rekt/resources/eventtype.WaitForEventType()`